v2/apiserver/config.go

```diff
@@ -244,16 +268,23 @@ func sessionsServiceConfig() (api.SessionsServiceConfig, error) {
 	}
 	thirdPartyAuthStrategy :=
 		os.GetEnvVar("THIRD_PARTY_AUTH_STRATEGY", thirdPartyAuthStrategyDisabled)
+	log.Println("THIRD_PARTY_AUTH_STRATEGY: ", thirdPartyAuthStrategy)
 	config.ThirdPartyAuthEnabled =
 		thirdPartyAuthStrategy != thirdPartyAuthStrategyDisabled
 	if config.UserSessionTTL, err =
 		os.GetDurationFromEnvVar("USER_SESSION_TTL", time.Hour); err != nil {
 		return config, err
 	}
+	log.Println("USER_SESSION_TTL: ", config.UserSessionTTL)
 	config.AdminUserIDs =
 		os.GetStringSliceFromEnvVar("ADMIN_USER_IDS", []string{})
+	log.Println("ADMIN_USER_IDS: ", config.AdminUserIDs)
 	config.GrantReadOnInitialLogin, err =
 		os.GetBoolFromEnvVar("GRANT_READ_ON_INITIAL_LOGIN", false)
+	if err == nil {
+		log.Println("GRANT_READ_ON_INITIAL_LOGIN: ", 
+		config.GrantReadOnInitialLogin)
+	}
```

Comment:
I leave it up to you whether you want to apply the same advice from [this comment](https://github.com/brigadecore/brigade/pull/1945/files?diff=split&w=0#r873838015) here. It seems less of an issue here because we're so close to the bottom of the function and the return that it's very clear to see what's going on here without any changes. Totally up to you whether to change it or leave it like this.